cpp/ps0004/Task4.cpp

```diff
@@ -14,6 +15,8 @@ int main()
     y=x;
     r=x;

```

Comment:
- [x] declared a variable to store the user input
- [x] chose a suitable data type for the variable
- [x] prompted user to enter a number
- [x] used `std::cin` in a right way to get user's input
⚠ try to use less memory. I can give you an example like `(char)x` or `(int)x` on the fly not in a variable